docs/language/values-and-types.mdx

```diff
@@ -1394,6 +1394,32 @@ booleans[0] = true
   //
   let containsKey42 = numbers.containsKey(42)
   ```
+- `cadence•fun forEachKey(function: ((K): Bool)): Void`
```

Comment:
```suggestion
- `cadence•fun forEachKey(_ function: ((K): Bool)): Void`
```

Reply:
Should we add labels to all the other function signatures too?

Reply:
This change effectively *removes* the argument label, as the parameter name is used as the argument label by default if none is specified. 

This is also how it is already documented, implemented, and used in the rest of the PR, e.g. https://github.com/onflow/cadence/pull/1992/files/887efe8b73cd24d620bcc6760dc36392b4639b2f#diff-c01765841e64b85306258f5ee4daff2bdc5b9e9e107167f55a6dba4162e8bdb8R4595, this was the only place where it was inconsistent.

Reply:
wait what lol, this whole time I thought the default if no label was specified was ArgumentLabelNotRequired. thanks for the heads up hahaha